solution/backend/cmcs_regulations/settings/base.py

```diff
@@ -79,6 +79,11 @@
 OIDC_OP_TOKEN_ENDPOINT = os.environ.get("OIDC_OP_TOKEN_ENDPOINT", None)
 OIDC_OP_USER_ENDPOINT = os.environ.get("OIDC_OP_USER_ENDPOINT", None)
 
+# BOX API SETTINGS
+BOX_CLIENT_ID = os.environ.get("BOX_CLIENT_ID")
+BOX_CLIENT_SECRET = os.environ.get("BOX_CLIENT_SECRET")
+BOX_REDIRECT_URL = os.environ.get("BOX_REDIRECT_URL")
+
```

Comment:
Note that this will look for the keys in the environment so we would need to update AWS with Paramstore value for BOX_CLIENT_ID, BOX_CLIENT_SECRET and BOX_REDIRECT_URL in addition if we choose JWT, we would also need to add the location of the RSA file, JWT key and passphrase